ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml

```diff
@@ -10,6 +10,7 @@ chain:
   - ref: cucushift-e2e
   - ref: openshift-extended-web-tests
   - ref: openshift-extended-test-supplementary
+  - chain: distributed-tracing-tests-common
```

Comment:
@jhou1 Adding the chain here means we will run those tests run as part of OCP tests. Is it expected ?

Reply:
If possible, we can have a meeting to discuss why we add this chain, by this time, we can consider this scenario with CY24 O1KR1.